src/Datadog.Trace.ClrProfiler.Managed/Integrations/AdoNet/DbCommandIntegration.cs

```diff
@@ -11,7 +11,7 @@ namespace Datadog.Trace.ClrProfiler.Integrations.AdoNet
     /// <summary>
     /// Instrumentation wrappers for <see cref="DbCommand"/>.
     /// </summary>
-    public static class AdoNetIntegration
+    public static class DbCommandIntegration
     {
         // TODO: rename to DbCommandIntegration
         private const string IntegrationName = "AdoNet";
```

Comment:
This should be renamed to "DbCommand", right? If we use this constant to check match against the integration being enabled.

Reply:
This is string is used when checking for configuration settings, so changing it is a breaking change for users. We should add a system where we can check multiple names so we can change the primary while keeping a fallback (`DbCommand` and `AdoNet`).

Reply:
I fixed the TODO comment in 5da0f543212a3bbf9f456c27e0a50273e3c931e9.